### ISSUE TITLE
Create Project in Mobile opens 404 for users not logged in.

### ISSUE THREAD

**Issue by Stephxdesigns** [unstructuredstudio/zubhub#821]
**Describe the bug**
Create Project in Mobile opens 404 for users not logged in instead of sign up/sign in

**To Reproduce**

1. Go to mobile landing page
2. Click on hamburger menu
3. Click on create project

**Expected behavior**
For better experience and to increase conversion rate users should be directed to the sign up/sign in page.

**Comment by coderatomy** [comment]
So basically @Stephxdesigns, we lacked a route protection to such routes that require authentication. If you don't mind, I request you help me with such routes. I have a PR [here ](https://github.com/unstructuredstudio/zubhub/pull/825) that fixes this. I have only added `/profile`, `/settings` and `/team`

cc @tuxology @srish

**Comment by srish** [comment]
@Stephxdesigns @coderatomy Ideally, a task like this is associated with a bigger task. We cannot just fix one route. Could you modify the task description, document what all routes need to be fixed, and invite designers to suggest if we should show an error message, redirect to log in, or take an alternative approach before improving the implementation?

**Comment by Stephxdesigns** [comment]
Hello @srish I'm a designer I think the best UX should be to redirect to login/sign. An error message would make users think something is broken and it doesn't give users a clear indication of what to do.
A redirection to login/sign up is most preferable it lets users know the action they need to take to create a project ( the "share your work" button on the landing page works this way for non logged in users)

And if an error message should be used it should clearly indicate to users that they need to sign up.

**Comment by Adaking-slack** [comment]
Yes, instead of a 404 error, it should be something like "unfortunately you can't access this page because you are not logged in /signed up" or  a more descriptive message describing why the error page is coming up, plus a CTA button that directs the user to sign up or log in

**Comment by jumokeadeoyo** [comment]
Based on @Stephxdesigns suggestion, It's more strategic to redirect users to the login/sign-up page directly when they click the create project button rather than an additional intermediate page with a call-to-action. This approach will reduce the number of screen the user will encounter, streamline the user journey, and enhance the user experience on our platform.

**Comment by Tegaphoenix** [comment]
@Stephxdesigns hi, @Adaking-slack raised this issue on web already and @srish already suggested the errors be collated and improvements improved across board. The mobile error was also spotted.
So if we we could all work togther to spot all errors and suggest UX and design improvements, we could get it to be a high priority issue which we can all contribute to working on.
See thread here: https://github.com/unstructuredstudio/zubhub/issues/789

Cheers

**Comment by Stephxdesigns** [comment]
Alright @Tegaphoenix I'd be happy to collaborate on this. What steps do we need to take

**Comment by Tegaphoenix** [comment]
I think we should locate and improve as many error pages as we can find.  Document our suggestions and wait for @srish review and feedback on monday.

**Comment by Stephxdesigns** [comment]
Alright @Tegaphoenix I'm available to collaborate on this maybe we can jump on a call with @Adaking-slack

**Comment by Stephxdesigns** [comment]
@Tegaphoenix For the desktop view some improvements were dropped on issue #789 I dropped some as well I don't know if we're to come up with a unified suggestion or different suggestions

**Comment by Tegaphoenix** [comment]
I am sure different suggestions will work, so there are amny options to choose from (I Think).

**Comment by jumokeadeoyo** [comment]
Hi @Stephxdesigns and @Tegaphoenix , I'm a designer and I would like to join in on this too.

**Comment by Adaking-slack** [comment]
Hi @Stephxdesigns @Tegaphoenix I am up to collaborate, what's our next line of action?

**Comment by Adaking-slack** [comment]
> @Tegaphoenix For the desktop view some improvements were dropped on issue #789 I dropped some as well I don't know if we're to come up with a unified suggestion or different suggestions

I suggest we document every single error page, discuss on how to improve both the ui and the ux, then we can each come up with different designs. So we have a unified problem with different solutions (hope that makes sense)

**Comment by Stephxdesigns** [comment]
I think we can jump on a call to to identify the error page and we can brainstorm and come up with one unified solution after all in the main issue srish said we should collaborate

**Comment by Adaking-slack** [comment]
ok sure

**Comment by Adaking-slack** [comment]
when are we hoping on the call?

**Comment by Stephxdesigns** [comment]
@Adaking-slack @Tegaphoenix @jumokeadeoyo  Will 6 pm WAT be a good time for you all?
I'm not available at the moment please send in your time suggestions

**Comment by Adaking-slack** [comment]
7:30?

**Comment by Stephxdesigns** [comment]
@Adaking-slack that works for me too

**Comment by coderatomy** [comment]
when is the date for this call @Stephxdesigns

**Comment by Stephxdesigns** [comment]
@coderatomy  Oct 7th 7:30 pm WAT

**Comment by coderatomy** [comment]
and how many hrs am I left with 😂??

**Comment by coderatomy** [comment]
It will be 9:30pm for me. Am attending. So are we using zoom, google  meet? I think you can create the meeting and share links here. We just join in at the time.

**Comment by Adaking-slack** [comment]
are we still having the meeting?

**Comment by Stephxdesigns** [comment]
@Adaking-slack yes please can you create a Google meet and send the link

**Comment by Adaking-slack** [comment]
okk

**Comment by Adaking-slack** [comment]
https://meet.google.com/dyp-msvb-ziw

**Comment by Adaking-slack** [comment]
Hi, we are still meeting today right?
